bin/mb-download

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bb
+
+(require '[babashka.curl :as curl])
+(require '[clojure.java.io :as io])
+(require '[clojure.repl :refer [pst]])
+(require '[clojure.string :as str])
+
+(defn url [version]
+  (str "https://downloads.metabase.com"
+       (when (str/starts-with? version "1") "/enterprise")
+       "/v"
+       version "/metabase.jar"))
+
+(defn download [version dir]
+  (io/copy
+    (:body (curl/get (url version) {:as :stream}))
+    (io/file (str dir "/" version ".jar"))))
+
+(defn download-jar! [version dir]
+  (try
+    (println (str "Downloading from " (url version)))
+    (download version dir)
+    (println (str "Downloaded " version ".jar to " dir))
+    (catch Exception e
+      (println (str "Error downloading version " version))
+      (pst e))))
+
+(defn without-slash [s] (str/replace s #"/$" ""))
+
+(defn -main [& [version dir]]
```

Comment:
What's the advantage of the vararg syntax here? (It's a bit more complicated here and at the only call site I can see.)

Reply:
I think it's just a quick way to call it, where if you give extra args they get ignored.